### PR TITLE
Add motivation to README

### PR DESCRIPTION
**Pull request by stijnvermeeren-swisstopo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,61 @@
 # Boreholes Data Extraction
 
-Boreholes Data Extraction is a data extraction pipeline that extracts depth layers with their corresponding material description from borehole profiles in form of pdfs.
+Boreholes Data Extraction is a pipeline to extract structured data from borehole profiles in PDF files. Extracted properties (currently coordinates, depths and associated material descriptions) are returned in JSON format, and (optionally) visualized as PNG images. This project was initiated by the Swiss Federal Office of Topography [swisstopo](https://www.swisstopo.admin.ch/), and is developed with support from [Visium](https://www.visium.ch/).
 
-## Limitations
+## Motivation
 
-Note that the project is under active development and there is no release to this date, nor has the project reached a maturity such that it could be used.
+The Federal Office of Topography swisstopo is Switzerland's geoinformation centre. The Swiss Geological Survey at swisstopo is the federal competence centre for the collection, analysis, storage and provision of geological data of national interest.
```
Comment (dcleres) [inline README.md:7]: Missing comma: 

The Federal Office of Topography swisstopo is Switzerland's geoinformation centre. The Swiss Geological Survey at swisstopo is the federal competence centre for the collection, analysis, storage, and provision of geological data of national interest.

File: README.md [hunk 2]
```diff
@@ -1,16 +1,61 @@
...
 
-Only German and French borehole profiles are supported as of now.
+Therefore, the goal of this project is to automate the extraction of structured data from borehole profiles as much as possible. As far as swisstopo is concerned, the use case is to integrate the data extraction pipeline with the application boreholes.swissgeol.ch ([Github](https://github.com/swisstopo/swissgeol-boreholes-suite)), where a user interface for efficient quality control of the automatically extracted data will also be implemented.
+
+All code and documentation is published in this Github repository as open source software. All other persons, companies or agencies who manage borehole data of their own, are welcome to use the data extraction pipeline on their own data and to contribute to the project with their own improvements/additions.
```
Comment (dcleres) [inline README.md:15]: Github -> GitHub

File: README.md [hunk 3]
```diff
@@ -1,16 +1,61 @@
...
+Below is a list of the most relevant properties for the extraction of structure data from borehole profiles. Properties that can be automatically extracted by the current pipeline are in **bold**. Properties for which the implementation of automatic extraction is actively being worked on, are in _italics_.
+
+* Metadata
+  * **Coordinates**
+  * _Date_
+  * _Drilling method_
+* Lithology / stratigraphy
+  * **Depths** (upper and lower bound of each layer)
+  * **Material descriptions** (as plain text)
+  * USCS classification, color, consistency, plasticity...
```
Comment (dcleres) [inline README.md:28]: Why not italic or bold?

File: README.md [hunk 4]
```diff
@@ -1,16 +1,61 @@
...
+  * Geological interpretations
```
Comment (dcleres) [inline README.md:29]: Why not italic or bold?

File: README.md [hunk 5]
```diff
@@ -1,16 +1,61 @@
...
+* Other
+  * _Hydrogeology (ground water levels)_
+  * Instrumentation
```
Comment (dcleres) [inline README.md:32]: Why not italic or bold?

File: README.md [hunk 6]
```diff
@@ -1,16 +1,61 @@
...
+  * Casing
```
Comment (dcleres) [inline README.md:33]: Why not italic or bold?

File: README.md [hunk 7]
```diff
@@ -1,16 +1,61 @@
...
+  * Borehole geometry
```
Comment (dcleres) [inline README.md:34]: Why not italic or bold?

File: README.md [hunk 8]
```diff
@@ -1,16 +1,61 @@
...
+  * ...
+
+
+### Related work
+
+Existing work related to this project is mostly focussed on the extraction and classification of specific properties from textual geological descriptions.  Notable examples include [GEOBERTje](https://www.arxiv.org/abs/2407.10991) (Belgium), [geo-ner-model](https://github.com/BritishGeologicalSurvey/geo-ner-model) (UK), [GeoVec](https://www.sciencedirect.com/science/article/pii/S0098300419306533) und [dh2loop](https://github.com/Loop3D/dh2loop) (Australia). The scope of this project is considerable wider, in particular regarding the goal of understanding borehole profiles in various languages and with an unknown layout, where the document structure first needs to be understood, before the relevant text fragments can be identified and extracted.
```
Comment (dcleres) [inline README.md:40]: Two spaces: `descriptions. Notable`

File: README.md [hunk 9]
```diff
@@ -1,16 +1,61 @@
...
+
+The automatic data extraction pipeline can be considered to belong to the field or [automatic/intelligent document processing](https://en.wikipedia.org/wiki/Document_processing). As such, it involves a combination of methods from multiple fields in data science and machine learning, in particular computer vision (e.g. object detection, line detection) and natural language processing (large language models, named entity recognition). Some of these have already been implemented (e.g. the [Line Segment Detector](https://docs.opencv.org/3.4/db/d73/classcv_1_1LineSegmentDetector.html) algorithm), others are planned as future work.
```
Comment (dcleres) [inline README.md:42]: belong to the field of

**Comment by dcleres** [comment]
Thank you for improving the README.md. It is much clearer now. 

Do you think we should also reference the two related label studio forks?

**Review by dcleres — APPROVED** [review]
It looks good to me. The content is good and very clear. I spotted a few typos that you could address before merging. Thank you for clarifying this part of the project!

File: README.md (hunk 1 above)
Comment (stijnvermeeren-swisstopo) [reply README.md:7]: That's not a universally accepted rule :) -> https://en.wikipedia.org/wiki/Serial_comma
However, I don't feel very strongly about it, and I can add the comma.

A general note: could you try using the "add a suggestion" Github feature for suggesting minor changes like this one? That would make it easier to find and to accept the suggested changes.
image

**Comment by stijnvermeeren-swisstopo** [comment]
> Do you think we should also reference the two related label studio forks?

Unless we decide that we actually want to maintain those forks and put them into production after all, I would not explicitly mention them here.

File: README.md (hunk 3 above)
Comment (stijnvermeeren-swisstopo) [reply README.md:28]: Not being actively worked on (at least not planned for the rest of the year). Same for the others.

**Pull request merged**